### ISSUE TITLE
Missing artifacts

### ISSUE THREAD

**Issue by mark-de-haan** [mcupak/beacon-java#14]
Im trying to set up a beacon for genetic data, so I cloned this repo and followed the README.
On mvn install:

```
Could not resolve dependencies for project com.dnastack:beacon-java-service:jar:1.0-SNAPSHOT: The following artifacts could not be resolved: 
org.ga4gh:beacon:jar:0.3.0, 
com.dnastack:beacon-adapter-api:jar:1
```
Seems org.ga4gh.beacon is no longer available on maven central.

Im just trying to setup a beacon interface on top of my genetic data, so far I haven't found a single documentation page that makes sense. Could you point me in the right direction?

**Comment by mcupak** [comment]
@Mark-de-Haan That's correct, the artifact is not in central (yet). You can try the [latest snapshot version](https://oss.sonatype.org/#nexus-search;gav~org.ga4gh~beacon~~~) on Nexus, but it's very new, and I'm not sure if it's completely compatible with this implementation. I should be able to try it out in the next few days. Alternatively, you can build it yourself from the [GitHub repository](https://github.com/ga4gh/beacon-team/).

Documentation is limited (we're working on it!), but there is some information [on the wiki](https://github.com/ga4gh/beacon-team/wiki). [This presentation](https://www.slideshare.net/MiroCupak/beacon-development) might be helpful too. 

Hope that helps!
